src/components/Services.jsx

```diff
@@ -40,6 +40,11 @@ export const Services = ({
             isInMaintenance={has(appsInMaintenanceBySlug, konnector.slug)}
           />
         ))}
+        {!hasConnections &&
+          suggestedKonnectors.length === 0 &&
+          candidatesConfig.konnectors.map(candidate => (
+            <CandidateServiceTile key={candidate.slug} konnector={candidate} />
+          ))}
```

Comment:
We now have a lot of conditions here and it becomes to be difficult to read and understand. Can we have something more declarative?